tests/test_gui.py

```diff
@@ -1405,3 +1405,73 @@ def test_filter_header(mock_db_with_data, qtbot):
 
     # Check that the filter icon is removed
     assert col_index not in header.filtered_columns
+
+
+### Test LogViewWindow
+LOG_WAIT_MS = 300  # a bit longer than polling interval (200ms)
+
+
+def test_logview_append_content(log_view_window, qtbot):
+    """Test appending new content to the log file."""
+    window, log_file_path = log_view_window
+    initial_content = log_file_path.read_text()
+    initial_stat = log_file_path.stat()
+    appended_text = "Line 3"
+
+    assert window.text_edit.toPlainText() == log_file_path.read_text()
+    assert initial_stat.st_size > 0
+    # Check internal state matches
+    assert window._last_size == initial_stat.st_size
+    assert window._last_mtime == initial_stat.st_mtime
+
+    with log_file_path.open("a") as f:
+        f.write(appended_text)
+
+    # Wait until the filesystem reflects the change
+    qtbot.waitUntil(lambda: log_file_path.stat().st_mtime > initial_stat.st_mtime, timeout=2000)
```

Comment:
Is this wait necessary? I'd assume that changes made on the same node, let alone in the same process, are visible as soon as the file is flushed/closed above.

Reply:
you're right it's probably not necessary, I removed it.